packages/cli/src/error_handler.ts

```diff
@@ -134,3 +156,10 @@ const errorHasCauseMessage = (
     typeof error.cause.message === 'string'
   );
 };
+
+/**
+ * If the parser finished processing arguments, attempt extracting subcommand information.
+ */
```

Comment:
Can we add tests for this in some command tests? E.g. throwing error when yargs hasn't finished parsing and we get empty from here?